querybook/server/lib/utils/token_utils.py

```diff
@@ -0,0 +1,51 @@
+from logic.user import get_user_by_id, update_user_properties
+from .crypto_utils import encrypt_token, decrypt_token, DecryptionError
+from app.auth.utils import AuthenticationError
+from lib.logger import get_logger
+
+LOG = get_logger(__file__)
+
+
+class TokenManager:
+    def __init__(self, token_type: str, encryption_key: bytes):
```

Comment:
can we just pass the secret direclty, instead of encoding it outside?

also can let the crypto utils accept the secret and do the encoding there.